code/modules/spells/ability_types/realized_aimed.dm

```diff
@@ -88,3 +88,229 @@
 		var/distance_decrease = get_dist(T, L) * 10
 		L.apply_damage((damage_amount - distance_decrease), WHITE_DAMAGE, null, L.run_armor_check(null, WHITE_DAMAGE))
 		new /obj/effect/temp_visual/revenant(get_turf(L))
+
+/obj/effect/proc_holder/ability/aimed/despair_swords
+	name = "Blades Whetted with Tears"
+	desc = "An ability that summons 2 swords to attack and slow nearby enemies. \
+		Each sword deals damage equal to 2% of the target's max HP as Pale, to a minimum of 40."
```

Comment:
Is this supposed to say maximum?

Reply:
No, it has a min damage of 40 so things with 800 hp take 40 pale instead of 16, but something with more than 2000 hp takes more. It's a realized ability and honestly doesn't tank TTK that much due to it's cooldown? iirc solely using it against an enemy with 1.0 Pale gives a TTK of about 5 minutes? Im down to tweak this if you find this egregious.

Reply:
No, you're right. I didn't read the code all that well when I made this review. It would be interesting if the swords spawned as mobs that fight on your behalf, though.